internal/descriptor/registry_test.go

```diff
@@ -779,3 +779,36 @@ func assertStringSlice(t *testing.T, message string, got, want []string) {
 		}
 	}
 }
+
+// TestVariableVerbparsing is a higher-level test than at the parser level of
+// the special logic around ensuring verbs are correctly parsed if the last
+// segment is a variable.
+func TestVariableVerbParsing(t *testing.T) {
+	reg := NewRegistry()
+	fd := loadFile(t, reg, `
+		name: 'example.proto'
+		package: 'example'
+		message_type <
+			name: 'B'
+			field <
+				name: 'name'
+				label: LABEL_OPTIONAL
+				type: TYPE_STRING
+				number: 1
+			>
+		>
+		service <
+			name: "BService"
+			method <
+				name: "Meth"
+				input_type: "B"
+				output_type: "B"
+				options <
+					[google.api.http] < post: "/v1/b/{name}:foo:bar" body: "*" >
+				>
+			>
+		>
+	`)
+
+	_ = fd
```

Comment:
Is the purpose of this test simply to check if the file loads, or do we want to make any assertions here?

Reply:
In the prior behavior such a file would cause the parser to error. It's meant as a higher level check against the same tests I put in the parser test file. I'll remove it as there are now higher level tests in mux_test.go as well.

Reply:
Removed!